src/main/java/org/folio/dew/batch/eholdings/EHoldingsJobConstants.java

```diff
@@ -9,6 +9,7 @@ public final class EHoldingsJobConstants {
   public static final String LOAD_FIELD_TITLE_NOTES = "titleNotes";
   public static final String LOAD_FIELD_PACKAGE_AGREEMENTS = "packageAgreements";
   public static final String LOAD_FIELD_TITLE_AGREEMENTS = "titleAgreements";
+  public static final String LOAD_FIELD_PACKAGE_PROVIDER = "providerLevelToken";
```

Comment:
```suggestion
  public static final String LOAD_FIELD_PROVIDER_TOKEN = "providerLevelToken";
```
Maybe better?

Reply:
Not sure. LOAD_FIELD_PACKAGE_PROVIDER using to load Provider if packageFields contains "providerLevelToken".
So, it answers whether we should load the provider for the package or not.